holmes/core/toolset_manager.py

```diff
@@ -302,16 +315,17 @@ def load_toolset_with_status(
                 )
             enabled_toolsets_from_cli.append(custom_toolset_from_cli)
         # status of custom toolsets from cli is not cached, and we need to check prerequisites every time the cli runs.
-        self.check_toolset_prerequisites(enabled_toolsets_from_cli)
+        self.check_toolset_prerequisites(
+            enabled_toolsets_from_cli, mute_log_status=True
+        )
 
         all_toolsets_with_status.extend(custom_toolsets_from_cli)
-        if using_cached:
-            num_available_toolsets = len(
-                [toolset for toolset in all_toolsets_with_status if toolset.enabled]
-            )
-            logging.info(
-                f"Using {num_available_toolsets} datasources (toolsets). To refresh: use flag `--refresh-toolsets`"
-            )
+        num_available_toolsets = len(
+            [toolset for toolset in all_toolsets_with_status if toolset.enabled]
+        )
+        logging.info(
+            f"Using {num_available_toolsets} datasources (toolsets). To view all available toolsets: run '/tools'. To refresh: use flag `--refresh-toolsets`"
```

Comment:
toolset_manager though used by both console and server, but I don't think it's good to import the very external library for SlashCommands